docs/_sidebar.md

```diff
@@ -0,0 +1,30 @@
+  - [**Getting Started**](/)
```

Comment:
I noticed there was a issue with the layout for the items for CLI Commands, please see link here -> https://share.getcloudapp.com/2Nu0vwA8